File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/notification-app-role-context_item.ts


import { INotificationAppRoleContext_Item } from './inotification-app-role-context_item';

export class NotificationAppRoleContext_Item implements INotificationAppRoleContext_Item {
  
	public NotificationID: number = 0;
	public NotificationMessageID: number = 0;
	public NotificationContextType: string = "";
	public NotificationContextTypeID?: number;
	public AppRole: string = "";
	public AppRoleID: number = 0;
	public Code: string = "";
  
}

File: GTTS/INL.MessagingService/Models/NotificationAppRoleContext_Item.cs

namespace INL.MessagingService.Models
{
    public class NotificationAppRoleContext_Item : INotificationAppRoleContext_Item
    {
        public long NotificationID { get; set; }
        public int NotificationMessageID { get; set; }
        public string NotificationContextType { get; set; }
        public int? NotificationContextTypeID { get; set; }
        public string AppRole { get; set; }
        public int AppRoleID { get; set; }
        public string Code { get; set; }
    }
}
